js/base/Exchange.js

```diff
@@ -912,7 +911,12 @@ module.exports = class Exchange {
                 }
                 resultingCurrencies.push (highestPrecisionCurrency);
             }
-            const sortedCurrencies = this.sortBy (resultingCurrencies, 'code');
+            sortedCurrencies = this.sortBy (resultingCurrencies, 'code');
+            this.baseQuoteCurrencies = this.indexBy (sortedCurrencies, 'code');
```

Comment:
this is the only actual addition in this PR. we store currencies in exchange-wide variable, which we can access in other methods.